### PR TITLE
FF87 removing -moz media features

### PR DESCRIPTION
**Pull request by rachelandrew**
Part of #2515 removes links to features, and adds a release note.

### CODE REVIEW

**Comment by mathewhodson** [comment]
I don't think this needs to be added to the release notes, because according to https://bugzilla.mozilla.org/show_bug.cgi?id=787521 these were disabled in web content a while ago.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Dammnit, I wrote a review of this one, but then somehow didn't end up submitting it. Sorry about that @rachelandrew

File: files/en-us/web/css/mozilla_extensions/index.html [hunk 1]
```diff
@@ -643,22 +643,10 @@ <h2 id="Media_features">Media features</h2>
 
 <div class="index">
 <ul>
- <li>{{CSSxRef("@media/-moz-mac-graphite-theme", "-moz-mac-graphite-theme")}}{{gecko_minversion_inline("1.9.2")}}</li>
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/mozilla_extensions/index.html:646]: Actually, let's not worry about this one.

**Review by chrisdavidmills — APPROVED** [review]
Done a bit of copy editing, and now this one is ready to go in.

**Pull request merged**